pkg/occlient/occlient.go

```diff
@@ -14,16 +14,18 @@ import (
 	"time"
 
 	"github.com/fatih/color"
-	"github.com/pkg/errors"
-	log "github.com/sirupsen/logrus"
-
 	servicecatalogclienset "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset/typed/servicecatalog/v1beta1"
 	appsclientset "github.com/openshift/client-go/apps/clientset/versioned/typed/apps/v1"
 	buildschema "github.com/openshift/client-go/build/clientset/versioned/scheme"
 	buildclientset "github.com/openshift/client-go/build/clientset/versioned/typed/build/v1"
 	imageclientset "github.com/openshift/client-go/image/clientset/versioned/typed/image/v1"
 	projectclientset "github.com/openshift/client-go/project/clientset/versioned/typed/project/v1"
 	routeclientset "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1"
+	templateclientset "github.com/openshift/client-go/template/clientset/versioned/typed/template/v1"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	types "k8s.io/apimachinery/pkg/types"
```

Comment:
imports should be logically grouped together

Reply:
that was auto goimports in goland :(